plugins/woocommerce/package.json

```diff
@@ -798,10 +798,8 @@
 				"node_modules/@woocommerce/e2e-core-tests/CHANGELOG.md",
 				"node_modules/@woocommerce/api/dist/",
 				"node_modules/@woocommerce/admin-e2e-tests/build",
-				"node_modules/@woocommerce/classic-assets/build",
```

Comment:
These removals seem to be unrelated to the code. Ignore if intentional.

Reply:
It seems these changes were a result of `pnpm install`. I’ve reverted them.